pkg/runtime/core/trainingruntime_test.go

```diff
@@ -19,6 +19,7 @@ package core
 import (
 	"context"
 	"fmt"
+	"k8s.io/apimachinery/pkg/util/intstr"
```

Comment:
Could you move this to the second group?

Reply:
Good catch, updated!